lib/active_insights.rb

```diff
@@ -7,7 +7,9 @@
 require "active_insights/engine"
 
 module ActiveInsights
-  mattr_accessor :connects_to, :ignored_endpoints, :enabled
+  mattr_accessor :connects_to, :ignored_endpoints, :enabled,
+                 :http_basic_auth_enabled, :http_basic_auth_user,
```

Comment:
what do you think about moving this into a new object? For example:
```
config.active_insights.http_basic_auth.enabled = true
```
Then we can remove a bunch of the controller stuff and move it into its own class:
```
class ActiveInsights::HttpBasicAuth
  attr_accessor :enabled, :user, :password 

  def enabled?
    enabled
  end

  ..
end

module ActiveInsights
  def self.http_basic_auth
    @http_basic_auth ||= HttpBasicAuth.new
  end
end